### DISCUSSION TITLE
Issue with desktop organizer application pausing wallpaper

### DISCUSSION THREAD

**Discussion by Gr1ffC0d3** [rocksdanister/lively#2410] [Q&A]
I have my live wallpaper set to pause when an app is in focus. I also have a desktop organization app(iTop Easy desktop.) The wallpaper pauses even when on the desktop because of this. Is there a way that I can tell the wallpaper to keep running if the foreground app is iTop Easy Desktop? 

I am on Windows 11 Pro.

**Comment by rocksdanister** [comment]
Navigate to 
`%LocalAppData%\Packages\12030rocksdanister.LivelyWallpaper_97hta09mmv6hy\LocalCache\Local\Lively Wallpaper` 
or for installer
`C:\Users\<UserName>\AppData\Local\Lively Wallpaper\AppRules.json`

Edit `AppRules.json` 
```
  {
    "AppName": "Program Name",
    "Rule": 1
  }
```
and restart Lively to apply new setting. 

This may or may not work depending on how the desktop organization program is built.
